### PR TITLE
configuration docs

### PR DESCRIPTION
**Pull request by ghedamat**

### CODE REVIEW

File: docs/v0.5.x/aliasing-plugins.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+          "s3-assets:bar-assets",
+          "s3-index",
+          "notify-slack"]
+```
+
+The name specified after the colon will be passed as the `name` property
+of the `options` argument to the addon's `createDeployPlugin` method. Plugins
+should use their name to retrieve configuration values. In this example,
+the foo-assets instance of the s3-assets plugin could have different configuration
+than the bar-assets instance does.
```
Comment (lukemelia) [inline docs/v0.5.x/aliasing-plugins.md]: This page should include a configuration file snippet showing where `plugins` goes and how the configuration would look.

File: docs/v0.5.x/configuration-overview.md [hunk 2]
```diff
@@ -1,3 +1,112 @@
...
+
+```javascript
+plugins: ["s3-assets", "s3-index", "notify-slack"]
+```
+
+Any plugins not included in the list will not have their hooks executed.
+
+It's also possible to [alias plugins](../aliasing-plugins).
+
+## Plugin developers:
```
Comment (lukemelia) [inline docs/v0.5.x/configuration-overview.md]: Let's keep this as part of an Authoring Plugins section

File: docs/v0.5.x/aliasing-plugins.md (hunk 1 above)
Comment (ghedamat) [inline docs/v0.5.x/aliasing-plugins.md]: done just below, let me know if that's what you were thinking

Comment (lukemelia) [inline docs/v0.5.x/aliasing-plugins.md]: Good, but let's show two redis "instances". Doesn't really make sense without that.

Comment (ghedamat) [inline docs/v0.5.x/aliasing-plugins.md]: lol right :)

Comment (ghedamat) [inline docs/v0.5.x/aliasing-plugins.md]: ok @lukemelia done, still a contrived example but should give the idea

**Comment by lukemelia** [comment]
:+1:

**Pull request merged**